src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -41,6 +45,29 @@
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
is it constant? see a message in slack

Reply:
@Sofasmile in slack? I don't see. Was it direct?

Reply:
yet it's done